### DISCUSSION TITLE
Can I change a package from .msi to .exe ?

### DISCUSSION THREAD

**Discussion by rwsdwatson** [microsoft/winget-pkgs#111154] [Q&A] [answered]
The new .exe format installer will upgrade the msi (it has an msi embedded). so can i just author a new version but change the yaml to be exe based not msi?
Note I do not want to host the .exe and msi packages separately any longer as that involves extra cost on the CDN and there were issues with users interacting with both types.
So I will no longer publish the msi only install package as it was only for winget.

Would it be better to submit a delete PR that removes this major version and then submit a PR to add the .exe version?

**Comment by mdanish-kh** [comment, marked as answer]
You can set `InstallerType: exe` and use AppsAndFeaturesEntries and set the InstallerType to msi to tell WinGet that it's an MSI installer bootstrapped into an EXE

```yaml
InstallerType: exe
AppsAndFeaturesEntries:
  - InstallerType: msi
```

**Reply by rwsdwatson** [reply]
Thanks I got that submitted correctly and its all working.
